pom.xml

```diff
@@ -7,7 +7,7 @@
 
 	<groupId>org.testcontainers</groupId>
 	<artifactId>jooq-testcontainers-codegen-maven-plugin</artifactId>
-	<version>0.0.2-SNAPSHOT</version>
+	<version>0.0.3</version>
```

Comment:
Currently the published version is 0.0.1 and the on-going changes are on 0.0.2-SNAPSHOT. If we merge and release the Liquibase support then it will be part of 0.0.2.